### DISCUSSION TITLE
right click

### DISCUSSION THREAD

**Discussion by ILaptev96** [PrismarineJS/mineflayer#2675] [Q&A]
The server has a lobby bot (NPCs) on it need to click the right mouse button to enter the game. How can I do this with mineflayer?

**Comment by amoraschi** [comment]
With `activateEntity`

**Reply by ILaptev96** [reply]
How do I detect the bot (NPCs) and pass it to activateEntity?

**Comment by rom1504** [comment]
Entities are in bot.entities

There is also entities in sight method
